DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageResourceTests.swift

```diff
@@ -38,7 +39,7 @@ final class UIImageResourceTests: XCTestCase {
         let image = UIImage(systemName: "circle.fill")!
         let imageResource = UIImageResource(image: image, tintColor: nil)
 
-        XCTAssertNotEqual(imageResource.calculateIdentifier(), "")
+        XCTAssertEqual(imageResource.calculateIdentifier().count, 32)
         XCTAssertGreaterThan(imageResource.calculateData().count, 0)
```

Comment:
how come these asserts are different.

Like 32 vs 40 characters and for 40 char there is contains check but not for 32 version?

Reply:
We append identifier of the color at the end if we recognise that the image should be tinted.

Thing is, icons are in a solid color, and the tint is only applied within the UIImageView.